index.html

```diff
@@ -58,6 +60,171 @@ <h1 class="header-title">
+                    <p class="work-header-item-two work-header-item-margin">2015</p>
+                </div>
+                <p class="work-pharagraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="work-tags">
+                    <a class="work-tag" href="#">html</a>
+                    <a class="work-tag" href="#">css</a>
+                    <a class="work-tag" href="#">javascript</a>
+                </div>
+                <button type="button" class="bg-white-button">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <h2 class="about-title">About <br> Myself</h2>
+        <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <div class="social about-social">
+            <a class="social-title" href="#">LET’S CONNECT</a>
+            <ul class="social-icons">
+                <li class="social-list">
+                    <a href="">
+                         <img src="./assets/img/icons/vector1.svg" alt="twitter" class="social-icon-item">
+                    </a>
+                </li>
+                <li class="social-list">
+                    <a href="">
+                         <img src="./assets/img/icons/vector2.svg" alt="twitter" class="social-icon-item">
+                    </a>
+                </li>
+                <li class="social-list">
+                    <a href="">
+                         <img src="./assets/img/icons/vector3.svg" alt="twitter" class="social-icon-item">
+                    </a>
+                </li>
+                <li class="social-list">
+                    <a href="">
+                         <img src="./assets/img/icons/vector4.svg" alt="twitter" class="social-icon-item">
+                    </a>
+                </li>
+                <li class="social-list">
+                    <a href="">
+                         <img src="./assets/img/icons/vector5.svg" alt="twitter" class="social-icon-item">
+                    </a>
+                </li>
+            </ul>
+            <button type="button" class="bg-white-button">Get My Resume</button>
+        </div>
+
+        <div class="about-tabs-wrapper">
+            <div class="tabs-title-wrapper tabs-title-first-wrapper">
+                <a class="tabs-title" href="#">Langguages</a>
+                <button class="tabs-button" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7" />
+                    </svg>
+                </button>
+            </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") is a list and should use the `ul` for grouping, while each of them should have a `li` tag.
- [ ] The technology in each category (e.g., "JavaScript", "HTML" and "CSS") should also be embedded inside the `Languages`

Reply:
:+1: This issue has been addressed successfully :heavy_check_mark: